omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_0/PatientAllergyResource2_0Test.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_0;
+
+import org.openmrs.Allergy;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.RestTestConstants2_0;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_0.PatientAllergyResource2_0;
+import org.junit.Before;
+
+public class PatientAllergyResource2_0Test extends BaseDelegatingResourceTest<PatientAllergyResource2_0, Allergy> {
+	
+	@Before
+	public void init() throws Exception {
+		executeDataSet(RestTestConstants2_0.ALLERGY_TEST_DATA_XML);
+	}
+	
+	@Override
+	public Allergy newObject() {
+		Allergy allergy = Context.getPatientService().getAllergyByUuid(RestTestConstants2_0.ALLERGY_UUID);

```

Comment:
Other examples do the same as I have done here. Besides when you new an object, the display test of the framework fails.